build-server/src/main/java/nl/tudelft/ewi/build/BuildServerModule.java

```diff
@@ -39,7 +43,9 @@ public ObjectMapper get() {
 			}
 		});
 		
-		bind(DockerManager.class).to(DockerManagerImpl.class);
+		bind(DockerClient.class).toInstance(DefaultDockerClient.fromEnv()
```

Comment:
The [`fromEnv()`](https://github.com/spotify/docker-client/blob/master/src/main/java/com/spotify/docker/client/DefaultDockerClient.java#L1137) takes the `DOCKER_HOST` and `DOCKER_CERT_PATH` environment values if available. These are commonly used in companion with boot2docker on Windows and OSX hosts. When the variables are not available it fails back on the defaults. For Linux this is the UNIX socket: `unix:///var/run/docker.sock`, and otherwise it uses the web port: `http://localhost: 2375`.